README.MD

```diff
@@ -0,0 +1,105 @@
+# PrismLang
+[![Build Status](https://api.travis-ci.org/EncryFoundation/PrismLang.svg?branch=master)](https://travis-ci.org/EncryFoundation/PrismLang)
+
+PrismLang is a Contract Oriented strong typed programming language with static type-checking developed for the [EncryCore](https://github.com/EncryFoundation/EncryCore).
```

Comment:
Prism is a contract-oriented statically typed scripting language developed especially for [EncryCore](https://github.com/EncryFoundation/EncryCore) protocol.